app/views/content/table_of_contents/_edit.erb

```diff
@@ -0,0 +1,84 @@
+<% editable = true %>
+<% start_depth = contents.first.try(:ordinals).try(:length) %>
+<% current_depth = start_depth %>
+
+<% contents.each do |content| %>
+  <%# render separate opening and closing tags around sections %>
+  <% (1..content.ordinals.length - current_depth).each do %>
```

Comment:
I pulled the code down to try to figure this out more. If it's the first resource in a section insert the <div> tags on top of it?